azurelinuxagent/common/protocol/hostplugin.py

```diff
@@ -50,6 +52,14 @@ def __init__(self, endpoint, container_id, role_config_name):
         self.role_config_name = role_config_name
         self.manifest_uri = None
 
+    @staticmethod
+    def is_default_channel():
+        return HostPluginProtocol._is_default_channel
+
+    @staticmethod
+    def set_default_channel(is_default):
```

Comment:
If we want to set this to True all the time(other than UT cases), do we need the argument?